test/integration/dvl.cc

```diff
@@ -181,6 +181,7 @@ class DopplerVelocityLogTest : public testing::Test,
     seabed->SetLocalPose(seabedPose);
     seabed->SetLocalScale(math::Vector3d(1e3, 1e3, 0.0));
     seabed->SetMaterial(sand);
+    scene->DestroyMaterial(sand);
```

Comment:
Shouldn't this be automatically handled when the reference count on the material goes to 0 or when the scene itself is destroyed?

Reply:
this call is not needed so removed.